### PR TITLE
Added option to added Rounded Capped PieChart

### PR DESCRIPTION
**Pull request by Begarudev**
enhacement asked in this issue #1700

### CODE REVIEW

File: lib/src/chart/pie_chart/pie_chart_data.dart [hunk 1]
```diff
@@ -31,6 +31,10 @@ class PieChartData extends BaseChartData with EquatableMixin {
     PieTouchData? pieTouchData,
     FlBorderData? borderData,
     bool? titleSunbeamLayout,
+    CapStyle? capStyle,
+    bool? shadow,
```
Comment (imaNNeo) [inline lib/src/chart/pie_chart/pie_chart_data.dart:35]: Isn't it better to use the [Shadow](https://api.flutter.dev/flutter/dart-ui/Shadow-class.html) object?

Something like what we have in the LineChart:
https://github.com/imaNNeo/fl_chart/blob/11d13a78e6a134aa0cfdcd8dfa88dceaf79a4033/lib/src/chart/line_chart/line_chart_data.dart#L360-L361

And this is how we paint it in the `drawBarShadow()` function:
https://github.com/imaNNeo/fl_chart/blob/11d13a78e6a134aa0cfdcd8dfa88dceaf79a4033/lib/src/chart/line_chart/line_chart_painter.dart#L906-L939

Comment (imaNNeo) [reply lib/src/chart/pie_chart/pie_chart_data.dart:35]: And if it is transparent (default value), we will not draw it.

**Comment by imaNNeo** [comment]
As I see, it also tries to fix #1699.
But I think it would be better if you create separate PRs for this specific case to follow it up.

**Comment by moritz157** [comment]
Any news on this? I would really like to be able to round caps in donut charts too and looking at this it looks like it's pretty much solved and just has to be merged (and maybe reorganized first), right?

**Comment by imaNNeo** [comment]
@moritz157 It seems the PR author is not available anymore. Also, the branch is outdated. So I think we have to close it.

**Comment by diegoveloper** [comment]
any chance to re-open and finish the implementation of this feature? :D

**Pull request closed without merging**